src-browserify/core.js

```diff
@@ -0,0 +1,62 @@
+(function (name, context, definition) {
+  if (typeof module != 'undefined' && module.exports) module.exports = definition()
+  else context[name] = definition()
+})('cartodb', this, function() {
+  var cdb = {};
+  cdb.VERSION = "3.15.8";
+  cdb.DEBUG = false;
+  cdb.CARTOCSS_VERSIONS = {
+    '2.0.0': '',
+    '2.1.0': ''
+  };
+  cdb.CARTOCSS_DEFAULT_VERSION = '2.1.1';
+
+  var Loader = require('./core/loader');
+
+  cdb.config = {};
+  cdb.core = {
+    Profiler: require('./core/profiler'),
+    util: require('./core/util'),
+    Loader: Loader
+  };
+  cdb.Image = require('./vis/image.js');
+  cdb.SQL = require('./api/sql');
+  cdb.Tiles = require('./api/tiles');
+
+  cdb.image = {};
+  cdb.geo = {};
+  cdb.geo.ui = {};
+  cdb.geo.geocoder = {};
+  cdb.ui = {};
+  cdb.ui.common = {};
+  cdb.vis = {
+    Loader: Loader
+  };
+  cdb.decorators = {};
```

Comment:
Yes, will look into that now that I work on the subsequent bundles.

On Tue, Oct 20, 2015 at 10:42 AM, Pablo Alonso <notifications@github.com>
wrote:

> In src-browserify/core.js
> <https://github.com/CartoDB/cartodb.js/pull/723#discussion_r42468994>:
>
> > +    Loader: Loader
> > +  };
> > +  cdb.Image = require('./vis/image.js');
> > +  cdb.SQL = require('./api/sql');
> > +  cdb.Tiles = require('./api/tiles');
> > +
> > +  cdb.image = {};
> > +  cdb.geo = {};
> > +  cdb.geo.ui = {};
> > +  cdb.geo.geocoder = {};
> > +  cdb.ui = {};
> > +  cdb.ui.common = {};
> > +  cdb.vis = {
> > +    Loader: Loader
> > +  };
> > +  cdb.decorators = {};
>
> Are you planning to move them to the "standard" bundle then? I think it
> makes sense.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/cartodb.js/pull/723/files#r42468994>.
>



-- 

*Nicklas Gummesson*, software engineer
*CartoDB* — Map Your World's Data

Live sessions to learn CartoDB <http://cartodb.com/webinars/>